api/lib/source/polkadotV0-source.js

```diff
@@ -17,6 +18,7 @@ class polkadotAPI {
     this.store = store
     this.fiatValuesAPI = fiatValuesAPI
     this.db = db
+    this.topVoters = []
```

Comment:
this should not be here but in the store

Reply:
as the source is not persisted

Reply:
oh ok. I thought it was once it is created? I can move it